src/zoo/owl_zoo_cmd.ml

```diff
@@ -5,6 +5,19 @@
 
 open Owl
 
+let _read_file f =
+  let ic = open_in f in
+  let n = in_channel_length ic in
+  let s = Bytes.create n in
+  really_input ic s 0 n;
+  close_in ic;
+  (s) |> Bytes.to_string
+
+
+let _strip_last_char str =
+  if str = "" then "" else
+  String.sub str 0 ((String.length str) - 1)
+
```

Comment:
what is the use case of this function?

Reply:
This function is solely for stripping the last character `\n` from the gist string that was read in from the `gist.id` file, since the string returned by my `_read_file` function always contains the `\n`.

Reply:
OK, then you can use `String.trim`. Or you can just use `Owl_utils.read_file ~trim:true` then take the first element in the array, this might be the best solution since `read_file` return an array of string (for each line).